conn_go19_test.go

```diff
@@ -21,9 +21,7 @@ func (s *connSuite) TestExecBuild19() {
 	}
 	for _, tc := range testCases {
 		result, err := s.conn.Exec(tc.query, tc.args...)
-		if !s.NoError(err) {
-			continue
-		}
+		s.Require().NoError(err)
```

Comment:
it is not equivalent statement, because `s.Require().NoError(err)` stops whole test on first fail.
```
if !s.NoError(err) {
      continue
``` will run all cases and reports summary about failed cases.